test/functional/test_framework/wallet.py

```diff
@@ -40,9 +40,20 @@ def generate(self, num_blocks):
             self._utxos.append({'txid': cb_tx['txid'], 'vout': 0, 'value': cb_tx['vout'][0]['value']})
         return blocks
 
-    def get_utxo(self):
-        """Return the last utxo. Can be used to get the change output immediately after a send_self_transfer"""
-        return self._utxos.pop()
+    def get_utxo(self, txid=''):
```

Comment:
```suggestion
    def get_utxo(self, *, txid=''):
```

please force named arguments for this

Reply:
done